feature/system/management/README.md

```diff
@@ -0,0 +1,146 @@
+### Initial Setup:
+
+*   Connect DUT port-1, 2, 3 and 4 to ATE port-1, 2, 3 and 4
+*   Create VRF "mgmt" on DUT
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/name = mgmt
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/route-distinguisher = 64512:100
+*   Create an IPv6 networks ```ateNet``` attached to ATE port-1, 2, 3 and 4
+*   Create a loopback interface "lo0" on DUT and assign it an IPv6 address
+    *   /interfaces/interface[name=lo1]/config/name = lo1
+    *   /interfaces/interface[name=lo1]/config/type = softwareLoopback
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/ip
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/prefix-length
+*   Configure the loopback interface to participate in the VRF "mgmt"
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/vrf/name = mgmt
+
+##### Configure linecard ports to ATE using BGP
+
+*   Configure IPv6 addresses on DUT and ATE ports 1 and 2. Configure them to participate in the VRF "mgmt"
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/ip
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/prefix-length
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/vrf/name = mgmt
+*   Configure IPv6 eBGP between DUT Port-1 <--> ATE Port-1 and DUT Port-2 <--> ATE Port-2 in VRF "mgmt"
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/global/config/as = 64512
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/global/config/router-id = <router_id>
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=1<neighbor_ip>]/config/peer-as = 64511
+    * /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/route-distinguisher = 64512:100 
+*   Set default import and export policy to ```ACCEPT_ROUTE``` for the eBGP sessions
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-import-policy
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-export-policy
+*   Advertise a default route from ATE to DUT throught both the BGP sessions
+*   Advertise the loopback interface from DUT to ATE through both the BGP sessions
+    *   /network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/prefix = <lo0 IPv6 Address>
+
+##### Configure linecard ports to ATE using VRRP
```

Comment:
I don't think we need VRRP for linecard port. 

cc: @rszarecki

Reply:
ATE does not support VRRP, removing it, I am using a line card port in absence of oob. Just to test the static route operation